api_server.py

```
@@ -549,4 +559,5 @@ def add_security_headers(response):
 """
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))
+    # Binding to 0.0.0.0 is required for containerized environments to allow external access.
+    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))  # nosec B104
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)